pontoon/teams/views.py

```diff
@@ -77,21 +77,21 @@ def ajax_projects(request, locale):
         .annotate(enabled_locales=Count('project_locale', distinct=True))
     )
 
-    no_visible_projects = (locale.project_set.visible().count() == 0)
+    locale_projects = locale.available_projects_list()
 
-    no_projects_to_request = (
-        Project.objects.visible().exclude(Q(locales=locale)).count() == 0
-    )
+    no_visible_projects = locale.project_set.visible().count() == 0
+
+    has_projects_to_request = projects.visible().exclude(locales=locale).count() > 0
```

Comment:
Why use `.visible()` here? Above you'll see that `projects` already includes `Project.objects.visible()`.

Reply:
Oh, so silly